tools/releaseBuild/azureDevOps/templates/upload.yml

```diff
@@ -39,6 +39,17 @@ steps:
     resourceGroup: '$(StorageResourceGroup)'
   condition: succeeded()
 
+- task: AzureFileCopy@4
+  displayName: 'upload signed min-size package to Azure - ${{ parameters.architecture }}'
+  inputs:
+    SourcePath: '$(System.ArtifactsDirectory)\signed\PowerShell-${{ parameters.version }}-win-${{ parameters.architecture }}-gc.zip'
+    azureSubscription: '$(AzureFileCopySubscription)'
+    Destination: AzureBlob
+    storage: '$(StorageAccount)'
+    ContainerName: '$(AzureVersion)-gc'
```

Comment:
Can you add a comment on what gc means or expand it to make it obvious?

Reply:
Add a comment in `packaging.psm1` and also updated the `displayName` to call out the suffix `-gc` is for 'Guest Config'.